guides/common/modules/proc_cloning-hosts.adoc

```diff
@@ -0,0 +1,13 @@
+[id="Cloning_Hosts_{context}"]
+= Cloning Hosts
+
+You can clone existing hosts.
```

Comment:
Some might say that here we should explain **Why** someone would want to do this.

You can clone hosts to ...

Not a blocker though :)

Reply:
I cannot come up with something useful. We can fix this in the future.